src/pyodide/internal/snapshot.ts

```diff
@@ -322,11 +334,20 @@ function memorySnapshotDoImports(Module: Module): string[] {
  * linear memory into MEMORY.
  */
 function makeLinearMemorySnapshot(Module: Module): Uint8Array {
-  const dsoJSON = recordDsoHandles(Module);
+  let json: SnapshotJson = recordDsoHandles(Module);
```

Comment:
Nit: It's not JSON yet, maybe call it `snapshotMeta` or just `metadata`

Reply:
Makes sense to me.